App/Containers/SplashScreen/SplashScreen.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Text, View,Button } from 'react-native'
```

Comment:
file formatting issues.

Reply:
I have formatted all the files now, so please review it once again!